products/sle-micro/main.pm

```diff
@@ -15,9 +16,27 @@ my $distri = testapi::get_required_var('CASEDIR') . '/lib/susedistribution.pm';
 require $distri;
 testapi::set_distribution(susedistribution->new());
 
-#######################
-# Testing starts here #
-#######################
+if (is_updates_test_repo && !get_var('MAINT_TEST_REPO')) {
```

Comment:
This section is desperately in need of comments

Reply:
Isn't similar code somewhere else? Maybe it can be put in a subroutine then.

Reply:
Would be even better

Reply:
The reason I didn't do that is that the current code relies on the variable `SCC_ADDONS` https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/products/sle/main.pm#L327
I want to avoid that variable since SLE Micro can't register against all those modules.
Also, the current code for SLE is not really necessary for SLE Micro, I have simplified according to the needs.

Reply:
> I want to avoid that variable since SLE Micro can't register against all those modules.
The `SCC_ADDONS` should contain only modules the system can register against.

Reply:
> The `SCC_ADDONS` should contain only modules the system can register against.

I know, and the current code is setting that variable which I don't want as SLE Micro will fail to register.

Reply:
OK, now I understand.

Reply:
After fixing the openQABot triggering SLE Micro updates, this is much simpler now.